src/ai/backend/manager/models/gql_models/vfolder.py

```diff
@@ -573,7 +573,7 @@ async def from_row(cls, graph_ctx: GraphQueryContext, vfolder_row: VFolderRow) -
             if isinstance(e, ModelCardProcessError):
                 error_msg = e.msg
             else:
-                error_msg = "Unknown error"
+                error_msg = repr(e)
```

Comment:
It needs to be checked if this is simply applied by sending the error as is. In the case of BackendAIError, the exception message can be passed, but if an internal query is being passed, it might not be appropriate.